### PR TITLE
Doc Messages, System Modules, Topics and Follow

### PR DESCRIPTION
**Pull request by FredPeal**

### CODE REVIEW

File: auth.md
Comment (SparoHawk) [inline auth.md:1]: `recieves` is correct. This is no a singular/plurar situation, it is significant to a third-person referral.

Please revert the text as it was.

File: users.md
Comment (SparoHawk) [inline users.md:1]: `recieves` is correct. This is no a singular/plurar situation, it is significant to a third-person referral.

Please revert the text as it was.

File: topics.md
Comment (SparoHawk) [inline topics.md:1]: **question(blocking):** Topics? Why is this here?

File: system-modules.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# System Module
+
+## Get System Modules
+
+This method enables you to retrieve all system modules and returns a SystemModuleInterface interface.
```
Comment (SparoHawk) [inline system-modules.md]: **suggestion(blocking):** Can we please have a link to the corresponding interface?

**Review by SparoHawk — CHANGES_REQUESTED** [review]

File: topics.md
Comment (FredPeal) [reply topics.md:1]: Because Max created the module topics in kanvas ecosystem and it is now in the SDK.

**Review by SparoHawk — APPROVED** [review]

**Pull request merged**